activecampaign.php

```diff
@@ -59,7 +59,7 @@ function activecampaign_form_source($settings, $form, $static = false) {
 		if ($static) {
 			$source .= "static=1&";
 		}
-		$source .= sprintf("id=%d", $form["id"]);
+		$source .= sprintf("id=%d&%s", $form["id"], strtoupper(uniqid()));
```

Comment:
OK, I get it now. Yeah, a unique id is what you want. You wouldn't ever want it to pull a cached version.